service/src/main/resources/api/service_openapi.yaml

```diff
@@ -2271,6 +2294,9 @@ components:
         description:
           description: A description of the workspace
           type: string
+        properties:
```

Comment:
Outside scope for this PR, but I wonder if we'll eventually need "add/delete/modify properties" endpoints instead overwriting existing ones.
OTOH callers can just read-modify-write here which should be good for now, and we shouldn't add APIs until we need them

Reply:
As I understand it, today these tend to be set as descriptions and not frequently changed. As you say, we'll see what the usage is.